### ISSUE TITLE
[ASTextNode] Truncation Renderer May Cause Crash When Creating Its Context

### ISSUE THREAD

**Issue by Adlai-Holler** [facebook/AsyncDisplayKit#1650]
This is a rare one that came up on dev (AsyncDisplayKit release tag 1.9.7.3). My first thought was a race condition or invalid object getting set for `truncationAttributedString` but I've confirmed that we never set that property outside of init and Swift doesn't really allow you to perform incompatible casts.

Unfortunately we didn't capture the context about specific node hierarchy/class/configuration. Sorry for having so little to go on but I've got 20

<img width="1440" alt="screen shot 2016-05-03 at 6-1 34 53 pm" src="https://cloud.githubusercontent.com/assets/2466893/15186226/0e24cdb4-1751-11e6-8d51-bb87f4eae337.png">

Sorry I can't dive deeper into this right now. If nobody can, then I will when I have time but any help would be appreciated!

**Comment by maicki** [comment]
Curious if this is related to the lack of thread safety in ASTextNode and will be resolved if we merge: #1749

@Adlai-Holler: Any thoughts on closing this in favor of #1749 and reopen it if it comes up again?

**Comment by Adlai-Holler** [comment]
Sold! @adcooley keep an eye out for this in Tripstr after #1749 is merged.
